src/app/app.component.css

```diff
@@ -0,0 +1,59 @@
+mat-toolbar-row {
+  background: #0070C0
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0070C0" to be "#0070c0" (color-hex-case)](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5648599)